src/Microsoft.Identity.Web/TokenCacheProviders/Distributed/MsalDistributedTokenCacheAdapter.Logger.cs

```diff
@@ -58,6 +58,12 @@ private static class Logger
                     LoggingEventId.MemoryCacheRead,
                     "[MsIdWeb] {CacheType}: {Operation} cacheKey {CacheKey} cache size {Size} ");
 
+            private static readonly Action<ILogger, string, string, Exception?> s_l1CacheNegativeExpiry =
+                LoggerMessage.Define<string, string>(
+                    LogLevel.Debug,
```

Comment:
More like a LogLevel.Warning or Error?

Reply:
i don't think this error is actionable by the service.

Reply:
They should log a bug on MSAL

Reply:
i did.